Source Main 5.2/source/NewUIMainFrameWindow.cpp

```diff
@@ -1832,6 +1832,12 @@ int SEASON3B::CNewUISkillList::GetHotKey(int iHotKey)
 
 int SEASON3B::CNewUISkillList::GetSkillIndex(int iSkillType)
 {
+    // special handling for skills with different skill id for the trigger
+    if (iSkillType == AT_SKILL_BLAST_HELL_BEGIN)
+    {
+        iSkillType = AT_SKILL_BLAST_HELL;
+    }
```

Comment:
Taking inspiration from the comment here:
https://github.com/sven-n/MuMain/issues/78#issue-2725294683
but with a slightly different approach